arviz/stats/stats.py

```diff
@@ -305,27 +307,26 @@ def _ic_matrix(ics, ic_i):
     return rows, cols, ic_i_val
 
 
-def hpd(ary, credible_interval=0.94, circular=False):
```

Comment:
I think it is still useful to have circular karg, it is different than multimodal right?